service/src/main/java/bio/terra/cbas/runsets/outputs/OutputGenerator.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.cbas.runsets.outputs;
+
+import bio.terra.cbas.model.WorkflowOutputDefinition;
+import java.util.List;
+import java.util.Map;
+import org.databiosphere.workspacedata.model.RecordAttributes;
+
+public class OutputGenerator {
+  public static RecordAttributes buildOutputs(
```

Comment:
It would be great to add tests for OutputGenerator similar to ones we have for [InputGenerator](https://github.com/DataBiosphere/cbas/tree/main/service/src/test/java/bio/terra/cbas/runsets/inputs).

Reply:
Added tests for OutputGenerator.